RecoTracker/TransientTrackingRecHit/python/TTRHBuilderWithTemplate_cfi.py

```diff
@@ -15,3 +15,7 @@
 from Configuration.Eras.Modifier_phase1Pixel_cff import phase1Pixel
 phase1Pixel.toModify(TTRHBuilderAngleAndTemplate, PixelCPE = cms.string('PixelCPEClusterRepair'))
 
+# uncomment these two lines to turn on Cluster Repair CPE
+from Configuration.ProcessModifiers.phase2_PixelCPEGeneric_cff import phase2_PixelCPEGeneric
+phase2_PixelCPEGeneric.toModify(TTRHBuilderAngleAndTemplate, PixelCPE = cms.string('PixelCPEGeneric'))
```

Comment:
please drop type specifications for all parameters which already exist.
This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.
==> ```PixelCPE = 'PixelCPEGeneric'```